### PR TITLE
Feature/toolbar

### PR DESCRIPTION
**Pull request by SergioEstevao**
Refs #55 

This PR implements a first version of the Toolbar for RichText. At the moment is only display the icons of the toolbar on the top of the block.

This is uses the same code that the web version uses, with only some small adaptions for native.

The buttons still don't propagate the actions to the Aztec component, that will be done in a separate PR to keep things simple.

### CODE REVIEW

**Comment by SergioEstevao** [comment]
This was done by the react-native cmd line tool to link the library. I can
replace it by hand... should I?

File: android/app/build.gradle [hunk 1]
```diff
@@ -158,6 +158,7 @@ repositories {
 }
 
 dependencies {
+    compile project(':react-native-svg')
```
[the remark this answers is not in the document]
Comment (mzorz) [reply android/app/build.gradle]: > This was done by the react-native cmd line tool to link the library. I can
replace it by hand... should I?

Yes, there shouldn't be  a problem doing that 👍

**Comment by SergioEstevao** [comment]
@mzorz and @hypest , tests fixed, android gradle file sorted. Can you please give it another look?

**Comment by hypest** [comment]
Headsup: tried to update to the current latest branch (https://github.com/WordPress/gutenberg/pull/9012) and a test fails (`src/store/reducers/test.js`).

**Pull request merged**